File: java/mohit/Trees-BST/RangeSumOfBST.java
static int sum = 0;
static int solve(TreeNode root, int low, int high)
{
    //CODE HERE
    if(root == null) return sum;

    if(low <= root.val && root.val <= high) sum += root.val;

    if(root.val > low) solve(root.left, low, high);

    if(root.val < high) solve(root.right, low, high);

    return sum;
}

File: Python/vedant_bhatnagar/DevsnestProblems_THA/Phase_2/Trees/rangeSumofBST.py
def solve(root, low, high):
    # CODE HERE
    q = deque([root])
    Sum=0
    while q:
        n = q.pop()
        if n.val >= low and n.val<=high:
            Sum+=n.val
        if n.left:
            q.appendleft(n.left)
        if n.right:
            q.appendleft(n.right)
    return Sum